src/System.Globalization.Extensions/tests/IdnMapping/IdnMappingGetAsciiTests.cs

```diff
@@ -83,24 +83,22 @@ public void GetAscii(string unicode, int index, int count, string expected)
             }
             Assert.Equal(expected, new IdnMapping().GetAscii(unicode, index, count));
         }
-        
+
         [Fact]
-        public void TestGetAsciiWithDot()
+        [PlatformSpecific(TestPlatforms.AnyUnix ^ TestPlatforms.OSX)]
+        public void TestGetAsciiWithDot_Succeeds()
```

Comment:
Yes, it was at the moment, I was just trying to nail it down.  I was assuming that one of the *nixs used in CI would throw, and then I could add that platform to the appropriate side.  They all passed the test, though.

Looking into it a little further, it looks like it was failing on Ubuntu 16(04 at least) and openSUSE, so I'm switching to a ConditionalFact for the moment.